CONTRIBUTING.md

```diff
@@ -62,28 +62,34 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the WG for resolution and that discussion and
 compromise among collaborators be the default resolution mechanism.
 
-# Becoming a Collaborator
+# Becoming a Member and Collaborator
 
-All contributors who land a non-trivial contribution should be on-boarded in a timely manner,
+To become a member of the Community Committee, start participating! Attend the bi-weekly meetings, 
+file issues and pull requests, and provide insight via GitHub. Request to become an Observer by 
+filing an issue. Once added as an Observer to meetings, we will track attendance and participation 
+for 3 months, as according to our governance guidelines for participation. If you meet the mininum 
+attendance and are participating, the CommComm will vote to add you as a member.
+
+All participatns added as members should be on-boarded in a timely manner,
 and added as a collaborator, and be given write access to the repository.
```

Comment:
Double `and` - not sure if this is part of the original commits or not, but would be helpful to fix.